src/exec/src/server.rs

```diff
@@ -330,28 +330,36 @@ fn rust_occlum_pal_create_process(
 
     let stdio_raw = Box::new(stdio);
     let mut libos_tid = 0;
-    let create_process_args = Box::new(occlum_pal_create_process_args {
+    let argv = Box::into_raw(cmd_args_array.into_boxed_slice());
+    let env = Box::into_raw(cmd_envs_array.into_boxed_slice());
```

Comment:
Using `Box` is unnecessary here; so is `into_raw` and `from_raw`. What you want is `Vec::as_ptr`.

```rust
let argv = cmd_args_array.as_ptr();
let env = cmd_envs_array.as_ptr();
```

Reply:
Thanks for your review comments. Code updated